Xamarin.PropertyEditing.Mac/PropertyList.cs

```diff
@@ -91,10 +91,9 @@ public IHostResourceProvider HostResourceProvider
 
 		public override void ViewDidChangeEffectiveAppearance ()
 		{
-			if (this.propertyTable == null)
-				return;
+			base.ViewDidChangeEffectiveAppearance ();
 
-			this.propertyTable.BackgroundColor = this.hostResources.GetNamedColor (NamedResources.PadBackgroundColor);
+			UpdateResourceProvider ();
```

Comment:
Should this method be renamed `AppearanceChanged` for consistency with every other implementation?